lexicon/providers/porkbun.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import requests
+from lexicon.exceptions import AuthenticationError
+from lexicon.providers.base import Provider as BaseProvider
+import json
+
+LOGGER = logging.getLogger(__name__)
+
+NAMESERVER_DOMAINS = ["porkbun.com"]
+
+
+def provider_parser(subparser):
+    """Return the parser for this provider"""
+    subparser.description = """
+        To authenticate with Porkbun, you need both an API key and a
+        secret API key. These can be created at porkbun.com/account/api .
+    """
+
+    subparser.add_argument("--api-key", help="specify API key for authentication")
```

Comment:
What about `--auth-key` and `--auth-secret` here?

Reply:
That works for me, I had just used the wording from the Porkbun API page itself. I'll make the modification in a bit

Reply:
Edited - I kept the "help" wording and internal wording the same, hope that's alright.